autoware_adapi_v1_msgs/perception/msg/PredictedPath.msg

```diff
@@ -0,0 +1,2 @@
+geometry_msgs/Pose[] path
+float64 confidence
```

Comment:
maybe 'builtin_interfaces/msg/Duration time_step' is missing?
https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_perception_msgs/msg/PredictedPath.idl#L11

Reply:
Thank you.
Added back the missing `time_step`
[a9781b5](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/25/commits/a9781b5de6e26b3285a4271f2cd08c59165f5f81)